scipy/stats/tests/test_distributions.py

```diff
@@ -5297,6 +5296,19 @@ def test_sf_isf(self):
         x1 = stats.nakagami.isf(sf, nu)
         assert_allclose(x1, x0, rtol=1e-13)
 
+    @pytest.mark.parametrize('nu', [1.6, 2.5, 3.9])
+    @pytest.mark.parametrize('loc', [25.0, 10, 35])
+    @pytest.mark.parametrize('scale', [13, 5, 20])
+    def test_fit(self, nu, loc, scale):
+        # The first tuple of the parameters' values is discussed in gh-10908
```

Comment:
Consider mentioning here that this is a regression test for gh-13396 and that 21 of the 27 test cases failed prior to that PR.

Reply:
Done in c3683c3.

Reply:
Done in 5ca7b1d2bdaf2255800ad9ea2ce7be9780fefe58.